src/main/java/by/itech/lab/supplier/constant/ApiConstants.java

```diff
@@ -2,6 +2,7 @@
 
 public interface ApiConstants {
 
```

Comment:
Use class as a container for constants, not an interface.

Reply:
+++